plugins/doc_fragments/attributes.py

```diff
@@ -18,6 +18,20 @@ class ModuleDocFragment(object):
     description: Can run in C(check_mode) and return changed status prediction without modifying target.
   diff_mode:
     description: Will return details on what has changed (or possibly needs changing in C(check_mode)), when in diff mode.
+  idempotent:
+    description:
+      - When run twice in a row with the same arguments, the second invocation indicates no change.
+      - This assumes that the system controlled/queried by the module has not changed in a relevant way.
```

Comment:
That would be one of the next steps ;-) I've started the PRs here (c.crypto, c.dns, c.docker) since I know the content of the collections pretty well and was able to add the attribute everywhere. My next step would be to discuss this a bit further (especially the question about having another attribute for whether/when changed parameters are updated, or merging that attribute with this one), and then starting to add it to c.g (since that's going to be a *lot* of work since it requires good knowledge of every module it is added to).